src/components/main/Main.scss

```diff
@@ -25,11 +25,15 @@
 @media screen and (min-width: $breakpoint-m) {
   .main-container {
     padding: $spacing-m;
+    padding-left: $spacing-m;
+    padding-right: $spacing-m;
```

Comment:
These two declarations are not needed anymore because the above shorthand. 
Maybe some copy-paste thingy from my suggestion 😬 😁

Reply:
True, for both :D